budgetportal/views.py

```diff
@@ -605,3 +607,34 @@ def category_fields(category):
         'url_path': category.get_url_path(),
         'description': category.description,
     }
+
+
+def about(request):
+    videos_file_path = str(settings.ROOT_DIR.path('_data/videos.yaml'))
+    about_date_file_path = str(settings.ROOT_DIR.path('_data/about.yaml'))
+    navbar_data_file_path = str(settings.ROOT_DIR.path('_data/navbar.yaml'))
+    env = 'production'
+    if settings.DEBUG:
+        env = 'development'
+    context = {
+        'page' : {
+            'layout' : 'about',
+            'data_key' : 'about'
+        },
+        'site' : {
+            'data' : {
+                'videos' : read_object_from_yaml(videos_file_path),
+                'about' : read_object_from_yaml(about_date_file_path),
+                'navbar' : read_object_from_yaml(navbar_data_file_path)
+            },
+            'latest_year' : '2019-20'
+        },
+        'jekyll' : {
```

Comment:
Since we're leaving jekyll behind, the right fix for this is using the django `debug` context variable rather than defining a jekyll-specific variable in the django view.

Reply:
okay, so we will use 'debug' variable

Reply:
yup, please go ahead with that.